### PR TITLE
Add bilingual feature

### PR DESCRIPTION
**Pull request by talisman6803**
## Pull request checklist

**Please check if your PR meets the following requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Tested with (`npm run dev`) and is working great
- [ ] Build tested with (`node build/build.js`) and no errors
- [ ] Double checked the two items above

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Documentation
- [ ] Other (describe): 

## Describe your Pull Request

Modify ``BigName.vue`` , ``Title.vue`` , ``Target.vue`` for bilingual features.

## Additional content

_Void_

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-<style>
-#app {
-  font-family: "Avenir", Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
-</style>
+<template>
```
Comment (Taewan-P) [inline src/App.vue:1]: Nothing has changed on this file but committed as changed. Plz fix this

File: src/components/Banner.vue [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-  border-radius: 0.571rem;
-  margin: 0 0.357rem 1.071rem;
-}
-.newyear {
-  display: block;
-  width: 100%;
-  border: 0 rgb(231, 223, 223);
-  border-radius: 0.571rem;
-}
+<template>
```
Comment (Taewan-P) [inline src/components/Banner.vue:1]: Same here. Is there any difference that I haven't recognized?

**Review by Taewan-P — CHANGES_REQUESTED** [review]
I've commented on two of the files that were notified as `changed`. Actually nothing did.
All of these files are being notified as `changed` even from the parts that are not changed at all.

I don't know why this commit is messed up, but as the VCS is considering all of these files has been changed, I can not accept this PR until this is fixed.

**Comment by talisman6803** [comment]
This PR will be closed for a new PR in the new Forked Repository.

**Pull request closed without merging**